packages/web/docs/src/pages/docs/management/members-roles-permissions.mdx

```diff
@@ -0,0 +1,207 @@
+import NextImage from 'next/image'
+
+# Member, Roles, and Permissions
+
+A Hive organization can have an unlimited amount of members. Each member can be assigned a single
+role. A role is a collection of permissions that grant access to performing actions within the
+organization.
+
+Organization owners or users with the necessary permissions can assign pre-defined member roles or
+create and assign custom member roles with specific permissions tailored to your teams workflows.
+
+## Managing Members
+
+Managing members, roles and permissions is done within the organizations `Member` section on the
+Hive Console dashboard.
+
+Organization owners or organization members with sufficient permissions can access this section.
+
+import membersTabImage from '../../../../public/docs/pages/management/members-roles-permissions/members-tab.png'
+
+<NextImage
+  alt="Organization Member Section"
+  src={membersTabImage}
+  className="mt-8 rounded-lg drop-shadow-md"
+/>
+
+## Member Roles
+
+Member roles specify the actions a organization member can perform within the organization. Each
+organization member has a member role assigned.
+
+### Predefined Member Roles
+
+Each organization comes with a set of predefined member roles ready to use and assign to
+organization members. These roles are read-only, permissions can not be added or removed.
+
+| Role Name | Description                                                                                                          |
+| --------- | -------------------------------------------------------------------------------------------------------------------- |
+| Admin     | Full access to everything within the organization.                                                                   |
+| Viewer    | Full read access to everything within the organization. The default role for users added/invited to an organization. |
+
+### Creating Member Roles
+
+Instead of using the predefined member roles you can create your own member roles. Managing member
+roles is done within the `Roles` tab within the member section.
+
+import createRole01StartImage from '../../../../public/docs/pages/management/members-roles-permissions/create-role-01-start.png'
+
+<NextImage
+  alt="Members of an Organization"
+  src={createRole01StartImage}
+  className="mt-8 rounded-lg drop-shadow-md"
+/>
+
+Click the `Create new role` button for opening the wizard.
+
+import createRole02WizardImage from '../../../../public/docs/pages/management/members-roles-permissions/create-role-02-wizard.png'
+
+<NextImage
+  alt="Member Role Creator"
+  src={createRole02WizardImage}
+  className="mt-8 rounded-lg drop-shadow-md"
+/>
+
+Choose a name and a description for your new role. Then select the permissions for that role.
+
+Some permissions depend on each other. E.g. you can not allow deleting a project without granting
```

Comment:
I'm not sure if we should enforce this. I think delete without read could be valid but might just not be as practical for the member who does the deleting. I.e. they could still call the delete API but couldnt see what they are deleting.
I think this decoupling is beneficial by simplifying our checks and the UI.

Reply:
So far this documentation is Dashboard UI facing, and there you cannot delete a project without seeing that.
I think what you are referring to is more relevant for permissions for API access tokens.

Reply:
Is it the case that checking off "project:delete" would force-check "project:view"? If not, I'm not sure this needs to be mentioned?

If it is, I'm still not sure it needs to be mentioned. Perhaps it can be viewed as a detail with explanation left to e.g. in-situ tooltip.